docs/pouchdb.md

```diff
@@ -0,0 +1,199 @@
+# `gpii.pouch`
+
+A Fluid component that wraps a PouchDB instance and exposes select methods from the underlying object.
+
+# Component Events
+
+| Event                        | Description |
+| ---------------------------- | ----------- |
+| `onAllDocsComplete`          | Fired when a call to `{that}.allDocs` (see below) is successfully completed. |
+| `onBulkDocsComplete`         | Fired when a call to `{that}.bulkDocs` (see below) is successfully completed. |
+| `onBulkGetComplete`          | Fired when a call to `{that}.bulkGet` (see below) is successfully completed. |
+| `onCompactComplete`          | Fired when a call to `{that}.compact` (see below) is successfully completed. |
+| `onDestroyPouchComplete`     | Fired when a call to `{that}.destroyPouch` (see below) is successfully completed. |
```

Comment:
Remove this event as it's no longer available with the removal of `destroyPouch()` from `pouchdb-common.js`.